src/components/signin/components/Signin.js

```diff
@@ -22,7 +22,7 @@ export default class Signin extends React.Component {
         const form = this.formRef.props.form;
         form.validateFields(err => {
             if (err) {
-                return;
+                return err;
             }
```

Comment:
I would rather you spelled out `err` as `error`

Reply:
Done